CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+
+ - Added support for k8s deployments
+ - Removed support for RC based refreshes
+
+# 0.0.5
```

Comment:
I had left the heading of the next version as a separate exercise - post merge - at tagging point